assets/cart-notification.js

```diff
@@ -1,80 +1,85 @@
 class CartNotification extends HTMLElement {
-  constructor() {
-    super();
+	constructor() {
+		super();
 
-    this.notification = document.getElementById('cart-notification');
-    this.header = document.querySelector('sticky-header');
-    this.onBodyClick = this.handleBodyClick.bind(this);
+		this.notification = document.getElementById('cart-notification');
+		this.header = document.querySelector('sticky-header');
+		this.onBodyClick = this.handleBodyClick.bind(this);
 
-    this.notification.addEventListener('keyup', (evt) => evt.code === 'Escape' && this.close());
-    this.querySelectorAll('button[type="button"]').forEach((closeButton) =>
-      closeButton.addEventListener('click', this.close.bind(this))
-    );
-  }
+		this.notification.addEventListener('keyup', (evt) => evt.code === 'Escape' && this.close());
+		this.querySelectorAll('button[type="button"]').forEach((closeButton) =>
+			closeButton.addEventListener('click', this.close.bind(this))
+		);
+	}
 
-  open() {
-    this.notification.classList.add('animate', 'active');
+	open() {
+		this.notification.classList.add('animate', 'active');
 
-    this.notification.addEventListener('transitionend', () => {
-      this.notification.focus();
-      trapFocus(this.notification);
-    }, { once: true });
+		this.notification.addEventListener(
+			'transitionend',
+			() => {
+				this.notification.focus();
+				trapFocus(this.notification);
+			},
+			{ once: true }
+		);
 
-    document.body.addEventListener('click', this.onBodyClick);
-  }
+		document.body.addEventListener('click', this.onBodyClick);
+	}
 
-  close() {
-    this.notification.classList.remove('active');
+	close() {
+		this.notification.classList.remove('active');
 
-    document.body.removeEventListener('click', this.onBodyClick);
+		document.body.removeEventListener('click', this.onBodyClick);
 
-    removeTrapFocus(this.activeElement);
-  }
+		removeTrapFocus(this.activeElement);
+	}
 
-  renderContents(parsedState) {
-      this.cartItemKey = parsedState.key;
-      this.getSectionsToRender().forEach((section => {
-        document.getElementById(section.id).innerHTML =
-          this.getSectionInnerHTML(parsedState.sections[section.id], section.selector);
-      }));
+	renderContents(parsedState) {
+		this.cartItemKey = parsedState.key;
+		parsedState.sections &&
```

Comment:
Just modified line 36 - checking for parsedState.sections. https://github.com/Shopify/dawn/issues/264.